daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -88,7 +89,11 @@ func init() {
 // the EC2 Instance Metadata Service.
 func New(ctx logger.Context) (logger.Logger, error) {
 	logGroupName := ctx.Config[logGroupKey]
-	logStreamName := ctx.ContainerID
+	logStreamName, err := loggerutils.ParseLogTag(ctx, loggerutils.DefaultTemplate)
```

Comment:
@samuelkarp that's exactly right, the idea is to set a template on the daemon for all new containers. 
I'll make sure to create a PR for the docs to show that tag is now allowed but `awslogs-stream` would overwrite the tag.